js/index.js

```diff
@@ -157,12 +334,12 @@ function renderProjectsData(data){
       //Setting up project's date
       projectDateContainer.classList.add("project-dates-container");
       projectDate.classList.add("emphasis-fact");
-      projectDate.innerText = projectDates[arrayCounter];
+      projectDate.innerText = projectDataMap.get(repository.name).dates;
 
       //Setting up project's image
       projectImageContainer.classList.add("project-image-container");
       projectImage.classList.add("project-image")
-      projectImage.setAttribute("src", `./images/${projectImages[arrayCounter]}`);
+      projectImage.setAttribute("src", `./images/projects/${projectDataMap.get(repository.name).image}`);
```

Comment:
Another case correction, just change `/projects/` to `/Projects/`

Reply:
I will take a look into that as well, thanks